### ISSUE TITLE
request to add 5g only and 4g only mode

### ISSUE THREAD

**Issue by gstjee** [libxzr/FivegTile#1]
Hi, Currently FiveG toggle network mode either in NR/LTE mode or LTE mode. As 5g network coverage is still quite weak as compare to 4g coverage so phone mostly remains on 4g network only even when NR/LTE mode is activated.

**So can you please add NR only mode and LTE only mode instead of NR/LTE mode and LTE only mode.**

It will be really helpfull for those users who has weak 5g signal but still want to force 5g only mode.

**Comment by v0idspace** [comment]
Id like this option as another quick settings tile , my usecase is while traveling via train even though 5g is available throughout it phone switches to 4g which has basically no bandwidth, manually forcing 5g only makes the internet fast again

**Comment by gstjee** [comment]
> Id like this option as another quick settings tile...

Yeah that would be much better solution. Users can add both tiles into QS and then can use them as per the their circumstances.

**Comment by rahulbag3** [comment]
Came here to talk about the same issue, soul reason to use this tile to force switch to perticular network 5g/NR i get decent speed in 5g at my place over 200mbps but the network is very poor i mean if i force enabled it from settings it never goes away but its weeker then LTE so it switches to that instead, after turning of the toggle it goes to "global" instead of going to "LTE or the previous settings" for some reason, so it means thorough out the process i already had to go to settings twice, with this tile or without its kinda same, its a request to work on this, it ll be a game changer for us users.
